sql/errors.go

```diff
@@ -576,6 +576,15 @@ var (
 
 	// ErrSpatialTypeConversion is returned when one spatial type cannot be converted to the other spatial type
 	ErrSpatialTypeConversion = errors.NewKind("Cannot get geometry object from data you send to the GEOMETRY field")
+
+	// ErrInvalidBinaryPrimaryKey is returned when attempting to create a primary key with a binary column
+	ErrInvalidBinaryPrimaryKey = errors.NewKind("invalid primary key on binary column '%s'")
```

Comment:
Message / name needs adjusting

Binary columns are fine, it's specifically blob / text types that can't be used in keys

Reply:
Same with one below